rrap_dg/initial_coral_cover/icc.jl

```diff
@@ -269,42 +326,32 @@ function downscale_icc(
 
     return target_init_cover
 end
-function downscale_icc(rrapdg_dpkg::String, target_cluster::String, output_path::String)::Nothing
-    rme_gpkg = load_rme_gpkg(rrapdg_dpkg)
-    rme_icc = load_rme_cover(rrapdg_dpkg, rme_gpkg)
 
-    target_gpkg = GDF.read(joinpath(rrapdg_dpkg, "spatial", "$(target_cluster).gpkg"))
+"""
+    downscale_icc(rme_dataset::String, target_gpkg::String, output_path::String)::Nothing
 
-    icc = downscale_icc(rme_icc, rme_gpkg, target_gpkg)
+Downscale initial coral covers from ReefMod Engine for a given spatial area and export to
+provided output path.
 
-    savecube(icc, output_path, driver=:netcdf)
+# Arguments
+- `rme_dataset` : Path to rrap-dg data package
+- `target_gpkg` : Path to geopackage file defining area of interest
+- `output_path` : Path to export to resulting netCDF to
+"""
+function downscale_icc(dataset::String, target_gpkg::String, output_path::String)::Nothing
+    gbr_gpkg = load_gbr_gpkg(dataset)
+    rme_icc = load_rme_cover(dataset, gbr_gpkg)
+    target_gpkg = GDF.read(target_gpkg)
+
+    icc = downscale_icc(rme_icc, gbr_gpkg, target_gpkg)
+
+    try
+        savecube(icc, output_path, driver=:netcdf)
```

Comment:
For the cyclone datacube I'm using the flag `overwrite=true`. Do you prefer with or without it?

Reply:
I think without to avoid accidents...